README.md

```diff
@@ -5,30 +5,45 @@
 [![Coverage Status](https://coveralls.io/repos/github/xaptum/ecdaa/badge.svg?branch=master)](https://coveralls.io/github/xaptum/ecdaa?branch=master)
```

Comment:
This commit (7930fc7) looks ok to me, but please remove the "WIP" prefix from the commit message.

Reply:
Yea, sorry, forgot to push the amended commit with the new message. Done now